src/main/java/sirius/tagliatelle/rendering/GlobalRenderContext.java

```diff
@@ -234,4 +288,53 @@ public void storeExtraBlock(String name, String contents) {
 
         extraBlocks.put(name, contents);
     }
+
+    /**
+     * Returns the current debug level for rendered contents.
+     */
+    public DebugLevel getSiriusDebugLevel() {
+        return this.debugLevel;
+    }
+
+    /**
+     * Sets the debug level for rendered contents
+     *
+     * @param debugLevel equivalent String to be resolved as a valid {@link DebugLevel}
+     */
+    public void setSiriusDebugLevel(@Nonnull String debugLevel) {
+        try {
+            this.debugLevel = DebugLevel.valueOf(debugLevel.toUpperCase());
+            if (this.debugLevel != DebugLevel.OFF) {
+                buffer.append(Strings.apply("<!-- SIRIUS Cookie %s=%s -->\n", GlobalRenderContext.SIRIUS_DEBUG_COOKIE, this.debugLevel.toString()));
```

Comment:
Does this work for other files than `.html.pasta` as the generated comment is always an html comment?

Reply:
Will remove that as the safest option. Even if it worked for JS, breaks CSS.